lib/rubocop/cop/rails/to_s_with_argument.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Identifies passing any argument to `#to_s`.
+      #
+      # @safety
+      #   This cop is marked as unsafe because it may detect `#to_s` calls
+      #   that are not related to activesupport implementation.
```

Comment:
"Active Support" is the proper name :-)

> The proper names of Rails components have a space in between the words, like "Active Support". `ActiveRecord` is a Ruby module, whereas Active Record is an ORM.

https://guides.rubyonrails.org/api_documentation_guidelines.html#wording

```suggestion
      #   that are not related to Active Support implementation.
```